main.tf

```diff
@@ -147,6 +179,38 @@ resource "aws_vpn_connection" "tunnel" {
   tunnel1_ike_versions = var.tunnel1_ike_versions
   tunnel2_ike_versions = var.tunnel2_ike_versions
 
+  dynamic "tunnel1_log_options" {
+    for_each = [var.tunnel1_log_options]
+
+    content {
+      dynamic "cloudwatch_log_options" {
+        for_each = tunnel1_log_options.value
+
+        content {
+          log_enabled       = cloudwatch_log_options.value.log_enabled
```

Comment:
Please apply the same fixes in all aws_vpn_connection resources.

Reply:
Sorry about that. Done.